.travis.yml

```diff
@@ -1,7 +1,4 @@
 sudo: false
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4"
```

Comment:
There is no real need to drop support for 4.x, provided that the user has at least v4.5. I did this because Travis supports only v4.4, according to this link
[Available versions](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Available-Versions)
And that would cause the test to fail, I think, because async-hook requires Node > 4.5.
I don't have Travis and I can't check that v4.5 is really unsupported by Travis, and Travis docs are not outdated.
I could just try and re-enable v4 and see if the build passes...